package.json

```diff
@@ -133,6 +133,11 @@
 					"order": 7,
 					"type": "string",
 					"description": "Specify a command that will be executed when dry run is completed"
+				},
+				"intuita.useFormatting": {
```

Comment:
`intuita.formatWithPrettier`, because "useFormatting" is a command, not a statement